dcpy/connectors/socrata/__init__.py

```diff
@@ -0,0 +1,23 @@
+import os
+import requests
+from typing import Literal
+
+SOCRATA_USER = os.getenv("SOCRATA_USER")
+SOCRATA_PASSWORD = os.getenv("SOCRATA_PASSWORD")
+
+
+def _simple_auth():
```

Comment:
can we move this and `_socrata_request()` to some utils ile?

Reply:
Yup